From 2ffb8088f24039aabef574b2680cb373fdb4604a Mon Sep 17 00:00:00 2001
From: Yuri Prezument <y@yprez.com>
Date: Tue, 23 Feb 2016 13:05:14 +0200
Subject: [PATCH] Remove failing test

Remove test_map_timeout_exception_handler_returns_false() test.
The functionality for this was removed in https://github.com/kennethreitz/grequests/commit/394cc
---
 tests.py | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/tests.py b/tests.py
index b3789e3..2b4ef0f 100644
--- tests.py
+++ tests.py
@@ -130,17 +130,6 @@ def test_map_timeout_no_exception_handler(self):
         self.assertTrue(responses[1].ok)
         self.assertEqual(len(responses), 2)
 
-    def test_map_timeout_exception_handler_returns_false(self):
-        """
-        if you don't want your exceptions to show up in the map result
-        """
-        def exception_handler(request, exception):
-            return False
-        reqs = [grequests.get(httpbin('delay/1'), timeout=0.001), grequests.get(httpbin('/'))]
-        responses = grequests.map(reqs, exception_handler=exception_handler)
-        self.assertTrue(responses[0].ok)
-        self.assertEqual(len(responses), 1)
-
     def test_map_timeout_exception_handler_no_return(self):
         """
         ensure default behaviour for a handler that returns None
